client/packages/system/src/Name/Components/ManufacturerSearchInput/ManufacturerSearchInput.tsx

```diff
@@ -17,10 +19,13 @@ export const ManufacturerSearchInput = ({
   width = 250,
   value,
   disabled = false,
-}: NullableNameSearchInputProps) => {
+  textSx = {},
+}: NullableNameSearchInputProps & {
+  textSx?: SxProps<Theme>;
```

Comment:
Contemplating passing in all the AutoComplete props in case we need... but stopped here